packages/react-atlas-core/src/avatar/Avatar.js

```diff
@@ -6,32 +6,64 @@ import themeable from 'react-themeable';
  *
  * **NOTE**: children will always take precedence over props passed into component.
  */
-const Avatar = ({children, icon, image, title, ...props}) => {
-  const theme = themeable(props.theme);
-  let kids = children;
-  if (React.Children.count(children) === 1 && typeof children === "string"){
-    kids = <span {...theme(4, 'letter')}>{children[0]}</span>;
-  }
+ class Avatar extends React.Component {
+   constructor(props) {
+     super(props);
 
-  let avatar = null;
+     let image;
 
-  if (typeof image === "string") {
-    avatar = <img src={image} title={title} {...theme(3, 'image')} />;
-  } else if (image){
-    avatar = image;
-  } else if (icon){
-    avatar = icon;
-  } else if (title) {
-    avatar = <span {...theme(2, 'letter')}>{title[0]}</span>;
-  }
+     if(this.props.image === undefined) {
+       if(this.props.defaultImage === undefined) {
```

Comment:
While `something === undefined` works most of the time, it doesn't check against someone doing something silly like `undefined = 'defined'` and whatnot. Generally you'll want to do `typeof something === 'undefined'`. It's also a good practice because if you get into the habit of `something === undefined` you might eventually do `something == undefined` and that matches differently.